libraries/libfc/include/fc/crypto/bls_private_key.hpp

```diff
@@ -28,6 +28,7 @@ namespace fc::crypto::blslib {
 
          bls_public_key     get_public_key() const;
          bls_signature      sign( const std::vector<uint8_t>& message ) const;
+         std::string        get_pop_str() const;

```

Comment:
Add comment or rename to `proof_of_possession_str()`. If adding comment also note this is in AffineLE format.